utils/das_data_sorter.py

```diff
@@ -21,6 +94,9 @@ def avg_Data(array):
 parser.add_argument("--output", help="Returns the filtered data", default="filtered_data.csv", action="store")
 parser.add_argument("--unit", help="Specify time units (seconds, s, or minutes, m)", default="seconds", 
                     choices=["seconds", "s", "minutes", "m"], action="store")
+
+parser.add_argument("--smooth", help="Smoothens data points by 3-point mean or median smoothing", 
```

Comment:
What happens if a user does not input a `--smooth` argument?

Reply:
I suppose in this case, the user would just want normal converted data. Some users might want this, so that they can know the exact values at certain points of time.